### DISCUSSION TITLE
Setup window menu in JS/TS

### DISCUSSION THREAD

**Discussion by abcdefghijorngarbosaxyz** [tauri-apps/tauri#7356] [Q&A] [answered]
Hello. I have two questions:

- Can we set up window menu in JS/TS instead of rust?

- And how do we style it?

In this image the app window theme is dark but it still has white background
![image](https://github.com/tauri-apps/tauri/assets/19945743/691b0535-5dc7-4953-91ee-ab90aed5e854)

**Comment by FabianLars** [comment, marked as answer]
> Can we set up window menu in JS/TS instead of rust?

No: https://github.com/tauri-apps/tauri/issues/6617

> And how do we style it?

I think v2 will offer a dark mode but everything else is beyond what the OS apis offer and require you to do it in your frontend instead.
